dill/_dill.py

```diff
@@ -603,40 +731,74 @@ def load_session(filename='/tmp/session.pkl', main=None, **kwds):
     if module is not _main_module:
         return module
 
-def load_session_copy(filename='/tmp/session.pkl', **kwds):
+def load_vars(
```

Comment:
The name is much better.  Also consider: `load_namespace` and `load_module_vars`.  Fundamentally, the difference here between `load_session` and `load_vars` is the return object... right? The former returns as a module (or main), and the latter as a dict.  So... they should be either very closely named, or this function should begin with an underscore and be available from `load_session` through the use of a keyword argument.

Reply:
Rewriting my comment: `load_module_vars()` is preferable as it's parameters may diverge from the ones from `load_module()` as the time goes. There's already a new `update` parameter for it. Plus its functionality is discoverable when tab-completing `dill.load_m...` (without disrupting the completion).

I like `load_namespace`, but it can trick users into thinking it returns a `types.SimpleNamespace` object or something else.

Reply:
> it can trick users into thinking it returns a types.SimpleNamespace object or something else.

Ah... you are right.  Hmm. `load_module_asdict`?  `load_asdict`? `load_dict`?   Hmm.... I actually think I like `load_vars` better than `load_module_vars`.  I'm not sure about what this should be called yet.

Reply:
My favorites, in no specific order, are `load_module_asdict`, `load_module_vars` and `load_vars`. The other seem either nonspecific or possibly misleading. Let's leave it hanging for a while.

A short description of what the function does is:

> "The equivalent of
> ```python
> lambda file: vars(load_module(file)).copy()
> ```
> but without touching the original module."

---
If you prefer... 😜 

> ```python
> lambda file: load_module(file).__dict__.copy()
> ```

Reply:
`load_module_asdict`: we have a winner then.  ...and the first short description is perfect.